extensions/interactions/rule_templates.json

```diff
@@ -42,16 +42,16 @@
   "Continue": {},
   "DragAndDropSortInput": {
     "IsEqualToOrdering": {
-      "description": "is equal to ordering {{x|ListOfSetsOfHtmlStrings}}"
+      "description": "is equal to ordering {{x|ListOfSetsOfTranslatableHtmlContentId}}"
```

Comment:
If we are pluralizing items within lists/sets, then should this be ListOfSetsOfTranslatableHtmlContentId**s**?

Ditto for SetOfTranslatableHtmlContentIds below.